telegram/ext/_jobqueue.py

```diff
@@ -758,3 +767,6 @@ def __eq__(self, other: object) -> bool:
         if isinstance(other, self.__class__):
             return self.id == other.id
         return False
+
+    def __hash__(self) -> int:
+        return hash(self.id)
```

Comment:
wait where is `self.id` defined? Don't see it in the `__init__` or as a property

Reply:
have a look at `__getattr__` :)

Reply:
ah yes, looks like I tunnel visioned a bit